### ISSUE TITLE
This design is ridiculously low contrast by default

### ISSUE THREAD

**Issue by kevinmarks** [milligram/milligram#57]
By making the body text color #606c76 you have a contrast ratio of 5.4 (dangerously close to the 4.5 minimum for WCAG AA), which you then reduce further by using font weight 300 by default from a very spindly font, Roboto. 
Body text should meet AAA (7.5 contrast ratio) at minimum; if it's going to be viewed  on a mobile screen with ambient light, you will fail to be legible very rapidly.
I'd suggest you adjust the default $color-secondary to something much darker - ideally #000 but if you want a slight tint, at least pick something that won't make mobile users in daylight or older people curse you as they try to read your text.

**Comment by cjpatoilo** [comment]
Hi @kevinmarks 
Thanks for the info. I'll review the info you passed. Thank you for your attention and for your willingness to help this project be even better.
